CHANGELOG.md

```diff
@@ -1,3 +1,12 @@
+## Master
+
+##### Bug Fixes
+
+* Send a body with the `PATCH` request to deprecate a pod.  
```

Comment:
Kinda too late to this one, but as a user this means nothing to an average user. The changelog shouldn't talk about implementation details if it doesn't need them but instead explain what user flow or user action is now fixed.